contrib/alpine/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Tim <timstanleydev@gmail.com>
+# Maintainer: Tim <timstanleydev@gmail.com>
+pkgname=ebusd
+pkgver=21.3
+pkgrel=0
+pkgdesc="Ebus daemon"
+url="https://github.com/john30/ebusd"
+arch="all"
+license="GPL-3.0-only"
+makedepends="argp-standalone cmake mosquitto-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/john30/ebusd/archive/refs/tags/v$pkgver.tar.gz"
+
+build() {
+	cmake -B build \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_BUILD_TYPE=MinSizeRel \
+		-DBUILD_TESTING=ON
+	cmake --build build
+}
+
+check() {
+	ctest --output-on-failure --test-dir build
+}
+
+package() {
+	DESTDIR="$pkgdir" cmake --install build
+}
+
+sha512sums="
```

Comment:
should be single line I'd say

Reply:
their code style is to have this on a new line